### PR TITLE
doc: sort function names

### PR DESCRIPTION
**Pull request by yuyi98**
This PR is to sort `function` and `method` names in doc command.

doc

### CODE REVIEW

File: vlib/v/ast/str.v [hunk 1]
```diff
@@ -22,7 +22,7 @@ pub fn (node &FnDecl) str(t &table.Table) string {
 		receiver = '($node.receiver.name ${m}$name) '
 	}
 	name := node.name.after('.')
-	f.write('fn ${receiver}${name}(')
+	f.write(' fn ${receiver}${name}(')
```
Comment (alexesprit) [inline vlib/v/ast/str.v]: Why did you add a space here?

Comment (yuyi98) [reply vlib/v/ast/str.v]: Just for beauty.

Comment (alexesprit) [reply vlib/v/ast/str.v]: This function is used by `fmt` module, that means `fmt` will insert all function signatures with a space prefixed. That's why tests for this PR are failed.

Comment (yuyi98) [reply vlib/v/ast/str.v]: Sorry, I will fix it.

**Comment by yuyi98** [comment]
@alexesprit   Thanks, That's right.

**Comment by medvednikov** [comment]
Nice, thanks! Something I wanted to do :)
